contrib/gitian-build.py

```diff
@@ -217,6 +217,11 @@ def main():
     args.windows = 'w' in args.os
     args.macos = 'm' in args.os
 
+    # Disable for MacOS if no SDK found
+    if args.macos and not os.path.isfile('gitian-builder/inputs/Xcode-12.1-12A7403-extracted-SDK-with-libcxx-headers.tar.gz'):
+        print('Cannot build for MacOS, SDK does not exist. Will build for other OSes')
+        args.macos = False
+
```

Comment:
19817: should drop this. we do not skip macos build if no sdk is found, we download sdk instead and continue (see line 77).